src/components/bc-button.ts

```diff
@@ -75,6 +81,14 @@ export class Button extends withTwind()(BitcoinConnectElement) {
   private _onClick() {
     launchModal();
   }
+
+  // Handle keyboard events for accessibility (Enter/Space key triggers click)
+  public override _handleKeydown(event: KeyboardEvent) {
```

Comment:
I am worried about using a key listener, why cannot this be tabbed to and then enter pressed like normal buttons?

Reply:
Removed as the included `bci-button` is in fact a button element and receives the events normally